package.json

```diff
@@ -5,11 +5,7 @@
   "license": "MIT",
   "workspaces": {
     "packages": [
-      "packages/*",
-      "examples/graphiql-webpack",
-      "examples/monaco-graphql-webpack",
-      "examples/monaco-graphql-react-vite",
-      "examples/cm6-graphql-parcel"
+      "packages/*"
```

Comment:
Some of the examples have readme files which instruct users to `yarn` from root to install deps. We should make sure those readme files get updated if necessary.

Reply:
+1 on this, would love if we'd adjust this before merging!